libmproxy/flow.py

```diff
@@ -667,6 +667,10 @@ def unload_script(self, script_obj):
             self.add_event("Script error:\n" + str(e), "error")
         self.scripts.remove(script_obj)
 
+    def reload_scripts(self):
```

Comment:
3) Subscribe to the script change signal in `__init__`. The event handler should call `self.masterq.put(("script_change", script))`.
4) Add a `handle_script_change` function, that once called, takes the script object and calls `script.reload()`.